examples/printerdemo_mcu/ui/common.slint

```diff
@@ -241,7 +249,7 @@ export CheckBox := Rectangle {
             clicked => { checked = !checked; }
         }
 
-        Text {
+        if text != "" : Text {
```

Comment:
Smells like a future compiler optimisation :-) (remove_useless_text next to remove_useless_rectangles). Probably super rare though.

Reply:
that's not an optimization, that's to remove a gap in the layout  (so we don't have spacing for nothing)